webui/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM free5gc-base-v3:latest AS builder
+FROM ubuntu:18.04
+
+LABEL description="Free5GC open source 5G Core Network" \
+    version="Stage 3"
+
+ENV F5GC_MODULE webui
+ENV DEBIAN_FRONTEND noninteractive
+ARG DEBUG_TOOLS
+
+# Install debug tools ~ 100MB (if DEBUG_TOOLS is set to true)
+RUN if [ "$DEBUG_TOOLS" = "true" ] ; then apt-get update && apt-get install -y vim strace net-tools iputils-ping curl netcat ; fi
+
+# Set working dir
+WORKDIR /free5gc
+RUN mkdir -p config/ ${F5GC_MODULE}/
+
+# Copy executable and default configuration 
+COPY --from=builder /free5gc/${F5GC_MODULE} ./${F5GC_MODULE}/
```

Comment:
If we want to run webui, we should also copy free5gc/webconsole/public folder for webui's frontend.